polaris-react/playground/Playground.scss

```diff
@@ -0,0 +1,103 @@
+@import '../src/styles/tokens';
```

Comment:
Note to self.. Do NOT commit Playground changes.

Committed to have access to the hosted storybook playground and give reviewers some reference examples.